src/workloads/query/LimitSkip.yml

```diff
@@ -0,0 +1,142 @@
+SchemaVersion: 2018-07-01
+Owner: "@mongodb/query-execution"
+Description: |
+  This workload measures performance for $limit and $skip when different constants are supplied as
+  arguments.
+
+Keywords:
+- limit
+- skip
+- cache
+- parametrization
+- classic
+- sbe
+
+GlobalDefaults:
+  Database: &Database test
+  Collection: &Collection Collection0
+  DocumentCount: &DocumentCount 1e6
+  Threads: &Threads 1
+  MaxPhases: &MaxPhases 6
+  LimitSkipNumber: &limitSkipNum {^Inc: {start: 1, step: 100}}
```

Comment:
Let's make it random and not incremental

Reply:
I thought of making it random initially, then realized, we may get inconsistent results across different runs. With ^Inc and a set number of "Repeat," we can be sure that this workload runs deterministically. Do you think this would matter? If not, then going down the random route is better.

Reply:
Okay, that makes sense. Thanks.

However, how would that work for cases where we have both $limit and $skip - it would mean that we will always have skip == limit and the query will return no results ever.

Maybe we can separate skip and limit num definitions and start limit at 100 and skip at 0?

Reply:
Done.